seqspec/seqspec_check.py

```diff
@@ -43,7 +43,7 @@ def validate_check_args(parser, args):
         else:
             print("\n".join(errors))
 
-    return
+    return len(errors)
```

Comment:
since errors is an error with the written spec and not the arguments i think this should be modified.